### PR TITLE
chore: add CHANGELOG entry for 1.15.0

### PR DESCRIPTION
**Pull request by imurchie**
## Proposed changes

CHANGELOG entry for Appium 1.15.0.

Cc: @dpgraham @umutuzgur @jlipps @mykola-mokhnach @KazuCocoa 

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by dpgraham** [comment]
```
IOS (XCUITest):

feat: bump generic "iphone simulator" device name for iOS 13.0 from iPhone 8 to iPhone X (#1068)
feat: Add support for `otherApps` capability on ios like we have on Android already (#988)
feat: add "mobile: activeAppInfo" feature (#1025)
feat: include mjpegScalingFactor as xcuitest-driver settings (#983)
feat: Add safariGarbageCollect cap (#986)
feat: add includeSafariInWebviews cap (#1060)
feat: add safariGlobalPreferences cap (#1057)
feat: remove idevicedate (#1042)
feat: remove accomodations for older xcodes (#981)
feat: remove using iproxy (#996)
feat: remove idevicesyslog dependency for real devices(#1000)
feat: remove idevicelocation (#1006)
feat: get only user apps in file-movement for performance aspect (#1014)
feat: Remove ifuse pull (#1019)
feat: remove iwdp related capabilities, objects (#1023)
feat: switch reduceMotion via settings API instead of update simulator pref (#1065)

fix: fix GET /timeouts by adding it to the list of non-proxied urls (#1067)
fix: use new remote debugger async execute functionality (#1063)
fix: perform validation of retrieved derived data path (#1058)
fix: uninstall wda with '.xctrunner' suffix for real device (#1052)
fix: start proxy before WDA to handle caching issues (#1053)
fix: mjpegstream logic for returning screenshots (#1039)
fix: call destroy on finish to handle node bug (#1040)
fix: handle wda launch being called multiple times (#999)
fix: the control flow of perf record (#1002)
fix: make "get active element" proxy (#1011)
fix: correct the afc path and handle the case of documents container type
fix: use new target manipulation in appium-remote-debugger (#1028)
fix: catches errors in Safari when no available web pages (#1031)
fix: print the timeout ms instead of the wrong vars
fix: push files one by one instead of parrallel (#1034)
fix: remove visibility check from webview. (#963)
fix: remove broken call to _getLatestWebviewContextForTitle_ (#1038)
fix: apply minor fixes to startPerfRecord invocation (#1036)
```

**Comment by dpgraham** [comment]
```
IOS (Webdriveragent) Probably can combine this with XCUITest

feat: Add a new endpoint for element rotation (#213)
feat: Make it possible to start a session without starting an app (#195)
feat: return processArguments in "mobile: deviceInfo" (#194)
feat: Add the necessary primitives to be able to automate split-screen apps (#209)
feat: Add some useful details to the device info output (#210)
feat: [Xcode 11] Call includingNonModalElements to get snapshot (#216)
feat: Consider active application based on the current screen element (#214)
feat: Make the coordinates of the screen point configurable (#215)
fix: Make screenshot taking work for below iOS 11 as well (#193)
fix: Allow setting compression quality and scaling factor for the mjpeg-stream (#196)
fix: Filter out system app when there are multiple active apps (#204)
fix: Do not break immediately if click on text view fails
fix: Tune the condition for input preparation
fix: Only accept preparation if keyboard focus is set
fix: Default to active application call if no session is provided to source API (#208)
fix: Do not throw exception if the app is not in foreground after the timeout
fix: Switch tap and tapByCoordinate to their native implemntations (#212)
fix: Fix the name of the extension method
fix: Tune tap point calculation logic (#217)
fix: Throwing excpetion when looking for active application only if element is null (#219)
fix: correctly calculate touch coordinates in landscape orientation for iOS 13.1 (#220)
```

**Comment by dpgraham** [comment]
feat: Add timezone and locale in deviceinfo (#465)
fix: send key multi byte string (#474)
fix: clipboard get/set contentType (#478)
fix: store curContext in createSession (#483)
fix: set value immediately, replace value (#484) 
fix: Always print the original stacktrace rather than the wrapped one (#488)
fix: raise an error for launch app and close app since they do not apply to espresso 
fix: Sync createSession parameters with W3C spec (#490)
fix: raise an error for reset as they do not apply to Espresso (#491)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,86 @@
...
+  * `idevicesyslog` [#1000](https://github.com/appium/appium-xcuitest-driver/pull/1000)
+  * `idevicelocation` [#1006](https://github.com/appium/appium-xcuitest-driver/pull/1006)
+  * `ifuse` [#1019](https://github.com/appium/appium-xcuitest-driver/pull/1019)
+  * `ios-webkit-debug-proxy` [#1023](https://github.com/appium/appium-xcuitest-driver/pull/1023)
+* Get only user apps in file-movement for performance aspect [#1014](https://github.com/appium/appium-xcuitest-driver/pull/1014)
+* Switch `reduceMotion` via settings API instead of update simulator pref [#1065](https://github.com/appium/appium-xcuitest-driver/pull/1065)
+* Add the necessary primitives to be able to automate split-screen apps [#209](https://github.com/appium/webdriveragent/pull/209)
+* Add a new endpoint for element rotation [#213](https://github.com/appium/webdriveragent/pull/213)
+* Add `processArguments` and other useful information in `"mobile: deviceInfo"` [#194](https://github.com/appium/webdriveragent/pull/194) [#210](https://github.com/appium/webdriveragent/pull/210)
+* Add the necessary primitives to be able to automate split-screen apps [#209](https://github.com/appium/webdriveragent/pull/209)
```
Comment (jlipps) [inline CHANGELOG.md]: this is duplicated from above

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,86 @@
...
+[#353](https://github.com/appium/appium-base-driver/pull/353) [#355](https://github.com/appium/appium-base-driver/pull/355)
+* Fix handling of timezone when getting the device data/time [#392](https://github.com/appium/appium-ios-driver/pull/392) [#557](https://github.com/appium/appium-android-driver/pull/557)
+* Fix handling of globally installed `opencv4nodejs` and `mjpeg-consumer` modules [#105](https://github.com/appium/appium-support/pull/105)
+
+#### IOS (XCUITest):
+* Support iOS 13/Xcode 11
+* Bump generic `"iphone simulator"` device name for iOS 13.0 from `iPhone 8` to `iPhone X` [#1068](https://github.com/appium/appium-xcuitest-driver/pull/1068)
+* Add support for `otherApps` capability on iOS like we have on Android [#988](https://github.com/appium/appium-xcuitest-driver/pull/988)
+* Add `"mobile: activeAppInfo"` feature [#1025](https://github.com/appium/appium-xcuitest-driver/pull/1025)
+* Settings
```
Comment (KazuCocoa) [inline CHANGELOG.md]: - `snapshotTimeout`  https://github.com/appium/WebDriverAgent/pull/181
- `useFirstMatch` https://github.com/appium/WebDriverAgent/pull/187
- `defaultActiveApplication `  https://github.com/appium/WebDriverAgent/pull/209
- `activeAppDetectionPoint `: https://github.com/appium/WebDriverAgent/pull/215  https://github.com/appium/WebDriverAgent/pull/218

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,86 @@
...
+  * `iproxy` [#996](https://github.com/appium/appium-xcuitest-driver/pull/996)
+  * `idevicesyslog` [#1000](https://github.com/appium/appium-xcuitest-driver/pull/1000)
+  * `idevicelocation` [#1006](https://github.com/appium/appium-xcuitest-driver/pull/1006)
+  * `ifuse` [#1019](https://github.com/appium/appium-xcuitest-driver/pull/1019)
+  * `ios-webkit-debug-proxy` [#1023](https://github.com/appium/appium-xcuitest-driver/pull/1023)
+* Get only user apps in file-movement for performance aspect [#1014](https://github.com/appium/appium-xcuitest-driver/pull/1014)
+* Switch `reduceMotion` via settings API instead of update simulator pref [#1065](https://github.com/appium/appium-xcuitest-driver/pull/1065)
+* Add the necessary primitives to be able to automate split-screen apps [#209](https://github.com/appium/webdriveragent/pull/209)
+* Add a new endpoint for element rotation [#213](https://github.com/appium/webdriveragent/pull/213)
+* Add `processArguments` and other useful information in `"mobile: deviceInfo"` [#194](https://github.com/appium/webdriveragent/pull/194) [#210](https://github.com/appium/webdriveragent/pull/210)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md]: process arguments are for sure not included into the info returned by this endpoint

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,86 @@
...
+* Consider active application based on the current screen element [#214](https://github.com/appium/webdriveragent/pull/214)
+* Make the coordinates of the screen point configurable [#215](https://github.com/appium/webdriveragent/pull/215)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md]: This and the previous one are parts of a single feature, which allows to automate split-screen applications mode on iOS

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,86 @@
...
+* Allow setting compression quality and scaling factor for `mjpeg-stream` [#196](https://github.com/appium/WebDriverAgent/pull/196)
+* Fix `GET` `/timeouts` [#1067](https://github.com/appium/appium-xcuitest-driver/pull/1067)
+* Switch `async execute` to not need CORS (works for real and simulated devices) [#1063](https://github.com/appium/appium-xcuitest-driver/pull/1063)
+* Fix uninstalling of `WDA` with '.xctrunner' suffix for real device [#1052](https://github.com/appium/appium-xcuitest-driver/pull/1052)
+* Fix caching issues [#1053](https://github.com/appium/appium-xcuitest-driver/pull/1053)
+* Fix `mjpegstream` logic for returning screenshots [#1039](https://github.com/appium/appium-xcuitest-driver/pull/1039)
+* Fix Node bug when ending session [#1040](https://github.com/appium/appium-xcuitest-driver/pull/1040)
+* Fix launching `WDA` multiple times [#999](https://github.com/appium/appium-xcuitest-driver/pull/999)
+* Fix "get active element" [#1011](https://github.com/appium/appium-xcuitest-driver/pull/1011)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md]: the fix about getting the list of active contexts

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,86 @@
...
+* Fix screenshot taking for iOS below 11 [#193](https://github.com/appium/WebDriverAgent/pull/193)
+* Fix Filter out system app when there are multiple active apps [#204](https://github.com/appium/WebDriverAgent/pull/204)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md]: this is also a part of the split-screen automation feature above

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,86 @@
...
+* Fix getting source outside of session [#208](https://github.com/appium/WebDriverAgent/pull/208)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md]: this should not be in the change list. This functionality was broken in between some PRs after the previous stable version has been released

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,86 @@
...
+* Fix to not throw exception if the app is not in foreground after a timeout [#211](https://github.com/appium/WebDriverAgent/pull/211)
+* Fix `tap` and `tapByCoordinate` to use their native implementations [#212](https://github.com/appium/WebDriverAgent/pull/212)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md]: ..since iOS13

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,86 @@
...
+* Fix tap point calculation logic [#217](https://github.com/appium/WebDriverAgent/pull/217)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md]: Also part of the feature above

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,3 +1,86 @@
...
+* Default Chromedriver version is `77.0.3865.40` [#143](https://github.com/appium/appium-chromedriver/pull/143)
+* Add server feature to enable automated Chromedriver downloads [#548](https://github.com/appium/appium-android-driver/pull/548)
+* New capabilities
+  * `waitForLaunch` [#556](https://github.com/appium/appium-android-driver/pull/556) [#327](https://github.com/appium/appium-uiautomator2-driver/pull/327)
+  * `ensureWebviewsHavePages` [#553](https://github.com/appium/appium-android-driver/pull/553)
+  * `enforceAppInstall` [#573](https://github.com/appium/appium-android-driver/pull/573)
+* Fix partial platform version matching [#567](https://github.com/appium/appium-android-driver/pull/567)
+
+#### Android (UiAutomator2)
+* Make it possible to start server without an app or a package id [#337](https://github.com/appium/appium-uiautomator2-driver/pull/337)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md]: XCUITest also got such possibility: https://github.com/appium/WebDriverAgent/pull/195

**Comment by mykola-mokhnach** [comment]
Important change: WDA, UIA2 server and Espresso Server now operate in W3C mode and only support requests and responses in format, which is defined by W3C

File: CHANGELOG.md (hunk 5 above)
Comment (imurchie) [reply CHANGELOG.md]: What do you mean? It has nothing to do with contexts.

File: CHANGELOG.md (hunk 3 above)
Comment (imurchie) [reply CHANGELOG.md]: Then we need to work on commit messages.

File: CHANGELOG.md (hunk 5 above)
Comment (mykola-mokhnach) [reply CHANGELOG.md]: oh, I checked the wrong link. all good

File: CHANGELOG.md (hunk 3 above)
Comment (mykola-mokhnach) [reply CHANGELOG.md]: commit name is fine, we are just talking about different endpoints. `processArguments` has been added to active app info endpoint

Comment (KazuCocoa) [reply CHANGELOG.md]: Yeah, xcuitest has similar name commands:

`mobile:activeAppInfo` => processArguments, environment arguments and its bundleid
`mobile:deviceInfo` => timezone etc

https://github.com/appium/appium/blob/master/commands-yml/commands/mobile-command.yml

**Review by KazuCocoa — APPROVED** [review]

**Comment by KazuCocoa** [comment]
https://github.com/appium/appium/pull/13276 is also introduced in 1.15.0

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,3 +1,89 @@
...
+* New capabilities
+  * `includeSafariInWebviews` [#1060](https://github.com/appium/appium-xcuitest-driver/pull/1060)
+  * `safariGlobalPreferences` [#1057](https://github.com/appium/appium-xcuitest-driver/pull/1057)
+* Remove some third-party dependencies
+  * `idevicedate` [#1042](https://github.com/appium/appium-xcuitest-driver/pull/1042)
+  * `iproxy` [#996](https://github.com/appium/appium-xcuitest-driver/pull/996)
+  * `idevicesyslog` [#1000](https://github.com/appium/appium-xcuitest-driver/pull/1000)
+  * `idevicelocation` [#1006](https://github.com/appium/appium-xcuitest-driver/pull/1006)
+  * `ifuse` [#1019](https://github.com/appium/appium-xcuitest-driver/pull/1019)
+  * `ios-webkit-debug-proxy` [#1023](https://github.com/appium/appium-xcuitest-driver/pull/1023)
```
Comment (umutuzgur) [inline CHANGELOG.md]: ios-deploy is also removed

**Pull request merged**